types/nightwatch/index.d.ts

```diff
@@ -841,33 +916,47 @@ export interface NightwatchLanguageChains {
     have: Expect;
     with: Expect;
     at: Expect;
+    does: Expect;
     of: Expect;
+    include(...data: any): Expect;
+}
+
+export interface NightwatchExpectMethods {
+    active: Expect;
+    deep: Expect;
+    count: Expect;
 }
 
 export interface NightwatchTestSettings {
     [key: string]: NightwatchTestSettingScreenshots;
 }
 
-export interface Expect extends NightwatchLanguageChains, NightwatchBrowser {
+export function globalExpect(...args: any): Expect;
+export interface Expect extends NightwatchLanguageChains, NightwatchExpectMethods {
     /**
      * Returns the DOM Element
      */
-    element(property: any): this;
+    element(property: string | Element): this;
 
     /**
      * These methods will perform assertions on the specified target on the current element.
      * The targets can be an attribute value, the element's inner text and a css property.
      */
-    equal(value: string): this;
-    equals(value: string): this;
+    equal(value: string | string[] | number | number[]): this;
+    equals(value: string | string[] | number | number[]): this;
     contain(value: string): this;
     contains(value: string): this;
     match(value: string | RegExp): this;
+    matches(value: string | RegExp): this;
     startWith(value: string): this;
     startsWith(value: string): this;
     endWith(value: string): this;
     endsWith(value: string): this;
 
+    cookie(name: string, domain?: string): this;
+    domProperty(propertyName: string): this;
+    elements(property: string | Element): this;
```

Comment:
`elements` does not support the same functions as `element` or `cookie`

I think this needs to be reworked to something like this to have _finer grained_ types for each section in the https://nightwatchjs.org/api/expect/ documentation:
```ts
export interface Expect {
    element(cssSelector: string): ExpectElement;
    elements(cssSelector: string): ExpectElements;
    url(): ExpectUrl;
    cookie(): ExpectCookie;
    // title, ...
}
export interface ExpectElement extends NightwatchLanguageChains<ExpectElement> {
    equal(value: string): this;
    equals(value: string): this;
    css(property: string, message?: string): this;
    enabled: this;

    // ... more "element" related functions
}
export interface ExpectElements extends NightwatchLanguageChains<ExpectElements> {
    count: this;
    equal(count: number): this;

    // ... more "elements" related functions
}
export interface ExpectUrl extends NightwatchLanguageChains<ExpectUrl> {
    contain(value: string): this;
    // ... more "url" related functions
}
export interface ExpectCookie extends NightwatchLanguageChains<ExpectCookie> {
    contain(value: string): this;
    // ... more "cookie" related functions
}

export interface NightwatchLanguageChains<T> {
    to: T;
    be: T;
    been: T;
    is: T;
    that: T;
    which: T;
    and: T;
    has: T;
    have: T;
    with: T;
    at: T;
    of: T;
    not: T;
    before(value: number): T;
    after(value: number): T;
    // ... more properties
}
```